### DISCUSSION TITLE
Error in normalizePath(path.expand(path), winslash, mustWork)

### DISCUSSION THREAD

**Discussion by yuliristiana** [teamalgoritma/community#767] [Q&A] [answered]
Ketika saya akan menjalankan membuat array terdapat muncul error sebagai berikut 
![Screenshot 2023-09-22 192032](https://github.com/teamalgoritma/community/assets/134671802/f119416f-ee0a-40b3-b228-eafc8322215b)

Saya sudah coba dari beberapa sumber dan coba untuk restart tapi masih mengalami muncul error yang sama seperti diatas.

**Comment by divakartika** [comment, marked as answer]
Dear @yuliristiana 

Boleh dikonfirmasi apakah berhasil ketika pembuatan model keras? Dapat dicek dengan menjalankan `model <- keras_model_sequential()`, jika sudah berhasil maka tidak muncul error.

**Comment by ramdhinafinita** [comment]
Halo @yuliristiana , untuk menjaga agar forum diskusi berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami.
